lib/membrane/compositor_element.ex

```diff
@@ -93,17 +99,19 @@ defmodule Membrane.VideoCompositor.CompositorElement do
   @impl true
   def handle_pad_added(pad, context, state) do
     position = context.options.position
+    timestamp_offset = context.options.timestamp_offset
```

Comment:
Does it work? Shouldn't that be rather `context.pads.pad.options.timestamp_offset`?

Reply:
It does. Here is how Membrane.Element.CallbackContext.PadAdded struct looks like in `handle_pad_added`:
```elixir
%Membrane.Element.CallbackContext.PadAdded{
  direction: :input,
  options: %{position: {0, 0}, timestamp_offset: 2},
  pads: %{
    {Membrane.Pad, :input, #Reference<0.726837243.177209346.24902>} => %Membrane.Element.PadData{
      accepted_caps: {Membrane.RawVideo, [pixel_format: :I420]},
      availability: :on_request,
      caps: nil,
      start_of_stream?: false,
      end_of_stream?: false,
      direction: :input,
      mode: :pull,
      name: :input,
      ref: {Membrane.Pad, :input, #Reference<0.726837243.177209346.24902>},
      options: %{position: {0, 0}, timestamp_offset: 2},
      pid: #PID<0.725.0>,
      other_ref: :output,
      input_queue: nil,
      demand: 4000,
      demand_mode: :auto,
      demand_unit: :buffers,
      other_demand_unit: :buffers,
      auto_demand_size: 4000,
      sticky_messages: [],
      toilet: nil,
      associated_pads: []
    }
  },
  playback_state: :stopped,
  clock: nil,
  parent_clock: #PID<0.717.0>,
  name: :compositor
}
```